tests/integration_test.py

```diff
@@ -606,7 +618,104 @@ def test_pca_awareness(params_integration, tmpdir):
     end_prep_b = sum([1 for a in model_b.pop.all_agents if a.prep.active])
 
     assert math.isclose(init_active_b, init_active_a, abs_tol=20)
-    assert (end_active_b - init_active_b) > (end_active_a - init_active_a)
+    assert math.isclose(
+        (end_active_b - init_active_b), (end_active_a - init_active_a), abs_tol=20
+    )
 
     assert math.isclose(init_prep_b, init_prep_a, abs_tol=20)
     assert (end_prep_b - init_prep_b) > (end_prep_a - init_prep_a)
+
+
+@pytest.mark.integration_stochastic
+def test_migration(tmpdir):
+    param_file = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)), "params", "multi_location.yml"
+    )
+
+    params = create_params(None, param_file, tmpdir)
+
+    path_a = tmpdir.mkdir("a")
+    path_a.mkdir("network")
+
+    model = TITAN(params)
+
+    west_agents = [a for a in model.pop.all_agents if a.location.name == "west"]
+    num_init_west_agents = len(west_agents)
+
+    assert math.isclose(num_init_west_agents, 40, abs_tol=15)
+
+    model.run(path_a)
+
+    # most west_agents should now be east
```

Comment:
This is unidirectional migration, right? Should it also check to make sure none of the east agents are now in the west?

Reply:
Not specifically, I don't think.  As over time west agents can migrate away and migrate back (except all those probs are 0 and east staying in east probability is very high).  So we could check east to west, but it seems maybe a bit redundant?